tools/depends/target/ffmpeg/FFMPEG-VERSION

```diff
@@ -1,5 +1,5 @@
 LIBNAME=ffmpeg
 BASE_URL=https://github.com/xbmc/FFmpeg/archive
-VERSION=3.1.6-Krypton
+VERSION=release/3.3-kodi
```

Comment:
Yep, either cmake bump specifies too high (ffmpeg master?) versions for libraries or Kodi ffmpeg repo was not updated. https://github.com/xbmc/xbmc/pull/11946/commits/ffd5da1fb74db639f582e37ba89e80b6876b0e82#diff-ec68d1dfffe81bf01733e80ff85956e1R37 says _avcodec_ver ">=57.89.100" while version here https://github.com/xbmc/FFmpeg/blob/release/3.3-kodi/libavcodec/version.h#L30 is lower.